CONTRIBUTING.md

```diff
@@ -24,7 +48,8 @@ This is a rough outline of what a contributor's workflow looks like:
 - Make sure commit messages are in the proper format (see below).
 - Push changes in a topic branch to a personal fork of the repository.
 - Submit a pull request to operator-framework/operator-lifecycle-manager.
-- The PR must receive a LGTM from two maintainers found in the MAINTAINERS file.
+- The PR must receive a LGTM from a reviewer found in the OWNERS file.
+- The PR must receive an APPROVAL from a approver found in the OWNERS file.
```

Comment:
I asked about this process in my first PR...

https://github.com/operator-framework/operator-lifecycle-manager/pull/1972#issuecomment-767931204

And I got this response:

https://github.com/operator-framework/operator-lifecycle-manager/pull/1972#issuecomment-768044688

So it might be helpful to clarify how approvals are handled.